dbt_copilot_helper/commands/copilot.py

```diff
@@ -423,7 +423,8 @@ def _cleanup_old_files(config, output_dir, env_addons_path):
 def get_env_secrets(app, env):
     """List secret names and values for an environment."""
 
-    client = boto3.client("ssm")
+    session = get_aws_session_or_abort()
```

Comment:
Again here we could use `load_application_with_environment(app, env)` then `application.environments.get(env).session.client('ssm')`

Reply:
I don't think there's any need to call load_application here as it would introduce some unnecessary calls to aws and make it more complicated to get the session back out.